### DISCUSSION TITLE
Can I run a callback on auto connect?

### DISCUSSION THREAD

**Discussion by tedawf** [wevm/wagmi#2491] [Q&A]
Hi, I know wagmi config can set autoConnect to true, which will try reconnect on load. Can I make wagmi call my function on auto connect successfully?

**Comment by SaadNiaziDev** [comment]
place all functions in onConnect() of useAccount hook. Functions will get call when it get reconnect. 
https://wagmi.sh/react/hooks/useAccount#onconnect-optional
